gateway/api/server.go

```diff
@@ -705,4 +706,9 @@ func (api *Api) buildRoutes(r *apiroutes.Router) {
 		r.AuthMiddleware,
 		apiserverconfig.GenerateApiKey,
 	)
+
+	r.POST("/searches",
```

Comment:
If this searches for connections similarities, I'd say this should belong to the connections entity in someway rather than being a top level thing like this

Reply:
nevermind, I get it now